src/httprpc.cpp

```diff
@@ -322,3 +322,8 @@ void StopHTTPRPC()
         httpRPCTimerInterface.reset();
     }
 }
+
+const std::set<std::string>& GetWhitelistedRpcs(const std::string& userName)
```

Comment:
Probably can use `unordered_set`, but maybe that's a refactor for `g_rpc_whitelist`...

Reply:
@luke-jr I think unordered_set will be slower in most cases & use more memory, hence choosing set.